src/libraries/System.Text.RegularExpressions/tests/RegexCultureTests.cs

```diff
@@ -34,6 +34,7 @@ public void CharactersComparedOneByOne_AnchoredPattern(string pattern, string in
         [InlineData(RegexOptions.IgnoreCase | RegexOptions.CultureInvariant)]
         [InlineData(RegexOptions.Compiled)]
         [InlineData(RegexOptions.Compiled | RegexOptions.IgnoreCase | RegexOptions.CultureInvariant)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/39480", TestPlatforms.Browser)]
```

Comment:
As discussed above this needs to be reverted to condition based on NotInvariantGlobalization.

Reply:
Thanks!